src/main/java/bio/terra/service/dataset/flight/ingest/DatasetIngestFlight.java

```diff
@@ -86,25 +93,64 @@ public DatasetIngestFlight(FlightMap inputParameters, Object applicationContext)
 
     if (cloudPlatform.is(CloudPlatform.AZURE)) {
       addStep(
+          // TODO: This still uses GCP under the hood. Is this correct?
           new AuthorizeBillingProfileUseStep(
               profileService, ingestRequestModel.getProfileId(), userReq));
```

Comment:
@nmalfroy Is it still the case that we need google billing under the hood even for Azure?

Reply:
Actually, we shouldn't be anymore.  Once the snapshot work is implemented, we should have a ticket to remove the requirements and checks for the GCP project on azure billing profile uses and I'd remove this then

Reply:
Is there a ticket for this work so that I can note it down in the code?

Reply:
I'm going to create it now. One sec

Reply:
Thanks!

Reply:
Filling out now but here's the link
https://broadworkbench.atlassian.net/browse/DR-2107